### PR TITLE
(TK-231) Return 200 resp iff all services are up.

### PR DESCRIPTION
**Pull request by aperiodic**
Change the status service's handler to return a 200 OK HTTP response if
and only if all the services being queried have the :is-running field of
their status set to :true. If any service has any other value for this
field, or is missing the field from that response entirely, return a 503
Service Unavailable response.

### CODE REVIEW

File: test/puppetlabs/trapperkeeper/services/status/status_service_test.clj [hunk 1]
```diff
@@ -151,6 +157,23 @@
...
               (json/parse-string (slurp (:body resp)))))))))
 
+(deftest status-code-test
+  (testing "returns 503 response code when a service is not running"
+    (with-status-service app
+      [bar-service fail-service]
+      (let [{:keys [status]} (http-client/get "http://localhost:8180/status/v1/services/fail")]
+        (is (= 503 status)))
+      (let [{:keys [status]} (http-client/get "http://localhost:8180/status/v1/services")]
+        (is (= 503 status)))))
```
Comment (cprice404) [inline test/puppetlabs/trapperkeeper/services/status/status_service_test.clj]: Could we add a test here that validates we get a 200 for `/status/v1/services/bar`?  (Assuming we agree that's the desired behavior :) )

**Comment by cprice404** [comment]
+1 other than maybe adding one more assertion to the test

File: test/puppetlabs/trapperkeeper/services/status/status_service_test.clj (hunk 1 above)
Comment (MSLilah) [inline test/puppetlabs/trapperkeeper/services/status/status_service_test.clj]: :+1: on this suggestion

**Comment by MSLilah** [comment]
I'm also :+1: barring the addition of another test assertion

**Comment by aperiodic** [comment]
@cprice404 @fpringvaldsen so updated.

**Pull request merged**